base/show.jl

```diff
@@ -1021,10 +1021,14 @@ end
 # If an object with this name exists in 'from', we need to check that it's the same binding
 # and that it's not deprecated.
 function isvisible(sym::Symbol, parent::Module, from::Module)
-    owner = ccall(:jl_binding_owner, Ptr{Cvoid}, (Any, Any), parent, sym)
-    from_owner = ccall(:jl_binding_owner, Ptr{Cvoid}, (Any, Any), from, sym)
+    isdeprecated(parent, sym) && return false
+    if isconst(from, sym) && isconst(parent, sym)
+        isdefined(from, sym) || return false
```

Comment:
Also need to check that `isdefined(parent, sym)` here. But is it really necessary now to do all of these lookups separately, instead of before where it could optimize doing the lookup of the Binding once and then doing all of the queries against that?

Reply:
I don't believe this change is necessary without the full partition, so I can revert it for now.